sdk/python/feast/repo_config.py

```diff
@@ -243,6 +243,8 @@ def __init__(self, **data: Any):
         self._online_store = None
         if provider == "expedia":
             self.online_config = data.get("online_store", "redis")
+            if self.online_config["type"] == "redis":
+                self.online_config["full_scan_for_deletion"] = True
```

Comment:
Do we want to do this if the Redis config [already defaults](https://github.com/ExpediaGroup/feast/blob/22bcbc49b4c76c1152e64da8f496026e34f21a23/sdk/python/feast/infra/online_stores/redis.py#L79-L80) `full_scan_for_deletion` to true?

[True is what triggers deletion.](https://github.com/ExpediaGroup/feast/blob/22bcbc49b4c76c1152e64da8f496026e34f21a23/sdk/python/feast/infra/online_stores/redis.py#L171-L173)